device-types/Quantum/Xcellis-WFExtender-Gen2.yaml

```diff
@@ -0,0 +1,42 @@
+---
+manufacturer: Quantum
+model: Xcellis Workflow Extender Gen2
+slug: quantum-xcellis-workflow-extender-gen2
+u_height: 1
+is_full_depth: true
+console-ports:
+  - name: ServicePort
+    type: rj-45
+power-ports:
```

Comment:
I'm guessing these are the controller devices, as these are child devices they don't have their own power ports.

Reply:
Ah okay, yes just convert them to `module-bays`